editor/src/clj/editor/outline_view.clj

```diff
@@ -517,9 +517,9 @@
 (defn- get-selected-node-id
   [^TreeView tree-view]
   (let [selection-model (.getSelectionModel tree-view)
-        selected-items (.getSelectedItems selection-model)
-        ^TreeItem selected-item (first selected-items)]
-    (item->node-id selected-item)))
+        selected-items (.getSelectedItems selection-model)]
+    (when-let [^TreeItem selected-item (first selected-items)]
+      (item->node-id selected-item))))
```

Comment:
To reproduce #11076, select the root node, and then use `Option+Click` to remove it from the current selection. Then click on the caret icon to collapse or expand a node that can be renamed, and the click on its label.